From 492f7947e52204545531ad4403c6527ad2c62d81 Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Mon, 2 May 2022 17:18:29 +0300
Subject: [PATCH 06/21] cmd: tlv_eeprom: do_tlv_eeprom: stop using non-api
 read_eeprom function

IN the scope of do_tlv_eeprom, the error-checking provided by the
read_eeprom function is not required.
Instead use the API function read_tlv_eeprom.

Signed-off-by: Josua Mayer <josua@solid-run.com>
Reviewed-by: Stefan Roese <sr@denx.de>
---
 cmd/tlv_eeprom.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cmd/tlv_eeprom.c b/cmd/tlv_eeprom.c
index acbd198d575..5e78a104902 100644
--- a/cmd/tlv_eeprom.c
+++ b/cmd/tlv_eeprom.c
@@ -434,7 +434,7 @@ int do_tlv_eeprom(struct cmd_tbl *cmdtp, int flag, int argc, char *const argv[])
 	// If no arguments, read the EERPOM and display its contents
 	if (argc == 1) {
 		if (has_been_read != current_dev) {
-			ret = read_eeprom(current_dev, eeprom);
+			ret = read_tlv_eeprom(eeprom, 0, TLV_INFO_MAX_LEN, current_dev);
 			if (ret) {
 				printf("Failed to read EEPROM data from device.\n");
 				return 0;
@@ -468,7 +468,7 @@ int do_tlv_eeprom(struct cmd_tbl *cmdtp, int flag, int argc, char *const argv[])
 	// Read the EEPROM contents
 	if (cmd == 'r') {
 		has_been_read = -1;
-		ret = read_eeprom(current_dev, eeprom);
+		ret = read_tlv_eeprom(eeprom, 0, TLV_INFO_MAX_LEN, current_dev);
 		if (ret) {
 			printf("Failed to read EEPROM data from device.\n");
 			return 0;
